### ISSUE TITLE
[SCREENREADER]: Decorative icons should be suppressed for screenreaders

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#3683]
## Description

**[SCREENREADER]: Decorative icons should be suppressed for screenreaders**

For screenreaders, decorative icons are generally suppressed, so that the inserted text won't be read by screenreaders if the font fails to render.

## Point of Contact

Jennifer

## Acceptance Criteria

As a screenreader user, I want to understand the content of the page without  insertion of random character strings so that I may know what is relevant for me.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination https://staging.va.gov/disability/view-disability-rating/rating/

## Steps to Recreate

1. Enter `https://staging.va.gov/disability/view-disability-rating/rating/` in browser
2. Start screenreading device listed in Environment
3. Using developer tools to simulate slow connectivity, so that font-awesome fails to load
4. Verify that the injected character does not render as an icon and is read by screenreader

## Possible Fixes (optional)

Add `aria-hidden="true" role="image"` attributes to the `<i>` element.
```html
 <i class="fas fa-medal vads-u-margin-right--0p5" aria-hidden="true" role="img"></i>

```

## WCAG or Vendor Guidance (optional)

[WCAG 1.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html) Content that is hidden from sighted users SHOULD also be hidden from screen reader users, except where the hidden content is intended only for screen reader users.
[Font Awesome and Accessibility](https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility)
[WEBAIM discussion re icons and screenreaders](https://webaim.org/discussion/mail_thread?thread=8268) "to help users who replace fonts you can add role="img" to the icons so that they may use this to exclude them from being replaced in their custom style sheet"

## Screenshots or Trace Logs

![rateddisability-icon.png](https://images.zenhubusercontent.com/5d6685f1a77bcf04870bc81c/ed1b0402-6dc4-4643-b22c-236f44c99091)

**Comment by jason-gcio** [comment]
This was addressed in this ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/4013
